pkg/restapi/kms/operation/models.go

```diff
@@ -11,7 +11,13 @@ type createKeystoreReq struct {
 }
 
 type createKeyReq struct {
-	KeystoreID string `json:"keystoreID"`
-	KeyType    string `json:"keyType"`
+	KeyType string `json:"keyType"`
+}
+
+type signReq struct {
+	Message string `json:"message"`
+}
+
+type lockReq struct {
```

Comment:
I think the suffix `Req` conveys this is a REST request, but a passphrase (and possibly a lock salt) are rather arguments to a request or tokens being sent along a request.

To me `signReq` and `createKeyReq` are not equivalent to passphrase.

Reply:
renamed to `lockArg`